test/verify/check-system-services

```diff
@@ -84,6 +84,16 @@ class TestServices(MachineCase):
 
         self.browser.wait_in_text(self.svc_sel(service), state_new)
 
+    def select_file_state(self, state):
+        self.browser.click("#services-dropdown-file-state")
+        self.browser.set_checked("fieldset label:contains('{0}') input".format(state), True)
+        self.browser.click("#services-dropdown-file-state")
```

Comment:
These two functions should also wait for the "chip" (is that the right name?) to appear.

Reply:
Done